packages/typespec-python/package.json

```diff
@@ -60,7 +60,7 @@
     "js-yaml": "~4.1.0",
     "semver": "~7.6.2",
     "tsx": "~4.19.1",
-    "@typespec/http-client-python": "~0.5.0",
+    "@typespec/http-client-python": "https://artprodcus3.artifacts.visualstudio.com/A0fb41ef4-5012-48a9-bf39-4ee3de03ee35/29ec6040-b234-4e31-b139-33dc4287b756/_apis/artifact/cGlwZWxpbmVhcnRpZmFjdDovL2F6dXJlLXNkay9wcm9qZWN0SWQvMjllYzYwNDAtYjIzNC00ZTMxLWIxMzktMzNkYzQyODdiNzU2L2J1aWxkSWQvNDQ1NDA5OS9hcnRpZmFjdE5hbWUvYnVpbGRfYXJ0aWZhY3RzX3B5dGhvbg2/content?format=file&subPath=%2Fpackages%2Ftypespec-http-client-python-0.5.0.tgz",
```

Comment:
It is not permitted to merge personal download link into main branch. Please keep it same with main branch.

Reply:
@msyyc this PR is just to check the effects of this upstream microsoft/typespec PR, that's why the download link is here

You do raise a good point though, where these PRs can be marked as draft PRs to ensure they don't get merged into main. I'm going to update the contributing to explicitly mark them as draft PRs.